docs/products/postgresql/howto/setup-logical-replication.rst

```diff
@@ -0,0 +1,174 @@
+
+Requirements
+------------
+
+The following are the requirements to setup a **logical replica** to Aiven for PostgreSQL
+
+
+* PostgreSQL version 10 or newer.
+* Connection between the source cluster's PostgreSQL port and Aiven for PostgreSQL cluster.
+* Access to an superuser role on the source cluster.
+* ``wal_level`` setting to ``logical`` on the source cluster. To verify and change the ``wal_level`` setting check :ref:`this section <PG MigrateWAL>`.
+
+.. Note::
+    If you are using an AWS RDS PostgreSQL cluster as source, the ``rds.logical_replication`` parameter must be set to ``1`` (true) in the parameter group.
+
+Set up the replication
+----------------------
+
+To create a logical replication, there is no need to install any extensions on the source cluster, but requires a superuser account.
+
+.. Tip::
+    The ``aiven_extras`` extension enables the creation of a publish/subscribe-style logical replication without a superuser account, and it is preinstalled on Aiven PostgreSQL servers. For more info on ``aiven_extras`` check the dedicated `GitHub repository <https://github.com/aiven/aiven-extras>`_. The following example will assume ``aiven_extras`` extension is not available in the source PostgreSQL database.
+
+This example assumes a source database called ``origin_database`` on a self-managed PostgreSQL cluster. The replication will mirror three tables, named ``test_table``, ``test_table_2`` and ``test_table_3``, to the ``defaultdb`` database on Aiven for PostgreSQL. The process to setup the logical replication is the following:
+
+1. On the source cluster, connect to the ``origin_database`` with ``psql``.
+
+2. Create the ``PUBLICATION`` entry, named ``pub_source_tables``, for the test tables::
+
+    CREATE PUBLICATION pub_source_tables
+    FOR TABLE test_table,test_table_2,test_table_3
+    WITH (publish='insert,update,delete');
+
+.. Tip::
+    In PostgreSQL 10 and above, ``PUBLICATION`` entries define the tables to be replicated, which are in turn ``SUBSCRIBED`` to by the receiving database.
+
+    When creating a publication entry, the ``publish`` parameter defines the operations to transfer. In the above example, all the ``INSERT``, ``UPDATE`` or ``DELETE`` operations will be transferred.
+
+3. PostgreSQL’s logical replication doesn’t copy table definitions, that can be extracted from the ``origin_database`` with ``pg_dump`` and included in a ``origin-database-schema.sql`` file with::
+
+    pg_dump --schema-only --no-publications \
+    SRC_CONN_URI                            \
+    -t test_table -t test_table_2 -t test_table_3 > origin-database-schema.sql
+
+4. The default ``postgres`` superuser :doc:`is not available in Aiven for PostgreSQL <../concepts/dba-tasks-pg>`, thus the user defined in the ``ALTER TABLE`` invocations within ``origin-database-schema.sql`` must be changed from ``postgres`` to the default ``avnadmin``.
```

Comment:
So do we need to show people how to do this? Or do they need to take care about the user if they run an ALTER TABLE command on the source database? I'm not sure if this belongs here in the setup steps.

Reply:
It was in the original set of tasks in the doc and I was questioning myself as well. I'll remove it